src/components/DrumLoopPlayer.js

```diff
@@ -1,26 +1,42 @@
 import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+
+import playbutton from '../images/play.svg';
+import pausebutton from '../images/pause.svg';
 
 export default function DrumLoopPlayer({
   startDrumLoop,
   getDrumLoop,
-  isPlayin,
+  recordingSrc
 }) {
+  const [isPlayin, setIsPlayin] = useState(false);
+  useEffect(() => {
+    if (recordingSrc !== '') {
+      setIsPlayin(false);
+    }
+  }, [recordingSrc]);
+  
   return (
     <DrumLoopContainer>
       <PlayPauseButton
         type="button"
-        onClick={startDrumLoop}
+        onClick={handleClick}
         aria-label="pause play"
       >
-        <img src={isPlayin} height="35px" width="35px" alt="play pause" />
+        <img
+          src={isPlayin ? pausebutton : playbutton}
+          height="35px"
+          width="35px"
+          alt="play pause"
+        />
       </PlayPauseButton>
       <DrumLoopLabel htmlFor="drum-loop-select">
         Choose a Drum Loop
       </DrumLoopLabel>
       <DrumLoopSelect
         name="drum-loops"
         id="drum-loop-select"
-        onChange={getDrumLoop}
+        onChange={onChange}
```

Comment:
Really clean code and cool project! I just have one little recommendation change onChange to a different name to stop confusion.